ci/dev/lint.sh

```diff
@@ -6,7 +6,7 @@ main() {
 
   eslint --max-warnings=0 --fix $(git ls-files "*.ts" "*.tsx" "*.js" | grep -v "lib/vscode")
   stylelint $(git ls-files "*.css" | grep -v "lib/vscode")
-  tsc --noEmit
+  tsc --noEmit --skipLibCheck
```

Comment:
Note: added this because TS was complaining about type name conflicts in `dom` and the `webworker` lib. I _think_ this is okay because there's no need for us to check library declaration files when we're linting/running tsc IMO.

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/3806031/108436927-941af980-7209-11eb-8ab2-f05c80e399e5.png">